kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver_with_constraints.h

```diff
@@ -2149,10 +2144,9 @@ class ResidualBasedEliminationBuilderAndSolverWithConstraints
 
         // Filling constant vector
         if (ComputeConstantVector) {
-            #pragma omp parallel for
-            for (int i = 0; i < static_cast<int>(BaseType::mEquationSystemSize); ++i) {
-                rConstantVector[i] = 0.0;
-            }
+            IndexPartition<std::size_t>(BaseType::mEquationSystemSize).for_each([&rConstantVector](std::size_t Index){
```

Comment:
Gcc 4.8.4 seems to have a problem here
~~~
/__w/Kratos/Kratos/kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver_with_constraints.h: In member function 'bool Kratos::ResidualBasedEliminationBuilderAndSolverWithConstraints<TSparseSpace, TDenseSpace, TLinearSolver>::ComputeConstraintContribution(typename Kratos::ResidualBasedEliminationBuilderAndSolverWithConstraints<TSparseSpace, TDenseSpace, TLinearSolver>::BaseType::TSchemeType::Pointer, Kratos::ModelPart&, bool, bool)':
/__w/Kratos/Kratos/kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver_with_constraints.h:2147:40: error: invalid declaration of 'Kratos::ResidualBasedEliminationBuilderAndSolver<TSparseSpace, TDenseSpace, TLinearSolver>::mEquationSystemSize'
             IndexPartition<std::size_t>(BaseType::mEquationSystemSize).for_each([&rConstantVector](std::size_t Index){
~~~

```suggestion
            IndexPartition<std::size_t>(mEquationSystemSize).for_each([&rConstantVector](std::size_t Index){
```

does this work @ashishdarekar? Can you first try locally?
If not then you could try with a temporary variable

Reply:
This is not working, `error: use of undeclared identifier 'mEquationSystemSize'`